### PR TITLE
feat: Add tour screen

### PR DESCRIPTION
**Pull request by joaopedrodcf**
This adds:
Tour screen where you can slide on the tours
Fixed TS problems with css properties form react-native
Added Stylelint for React-native
Eslint now fixes .prettierrc.js and .stylelint.js
Improved Styled components linter using now the new styled property that contains the theme properties. Example `import { styled } from '../../../../styles';`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,5 +60,13 @@ Your APK is now on `wanderlust\android\app\build\outputs\apk\debug` path.
 
 ## Known Issues
 
-* `Loading dependency graph, done.` - Use Node LTS 
-* `Keysotre file ...debug.keystore not found for signing config 'debug'` - [[Fixed #14](https://github.com/wetrip/wanderlust/pull/14)]
+-   `Loading dependency graph, done.` - Use Node LTS
+-   `Keysotre file ...debug.keystore not found for signing config 'debug'` - [[Fixed #14](https://github.com/wetrip/wanderlust/pull/14)]
+
+# just click enter in every part and when it says if you want to save say yes
```
Comment (NelsonPRSousa) [inline README.md]: Was the # a typo?

File: src/components/screens/tours/tour-item/TourItem.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                    </TourDescriptionTextWrapper>
+                    <BottomRow>
+                        <DefaultText
+                            size={theme.metrics.largeSize}
+                            color={theme.colors.textColor}
+                            weight={800}
+                        >
+                            {`${numberDestinations} ${
+                                numberDestinations === 1
+                                    ? 'Destination'
```
Comment (NelsonPRSousa) [inline src/components/screens/tours/tour-item/TourItem.tsx]: We should import `utils/i18n.tsx` to show text on our components

File: README.md (hunk 1 above)
Comment (joaopedrodcf) [reply README.md]: Yeah typo, changed

File: src/components/screens/tours/tour-item/TourItem.tsx (hunk 2 above)
Comment (joaopedrodcf) [reply src/components/screens/tours/tour-item/TourItem.tsx]: Yeah sure! Gonna redo that part ty

File: src/components/screens/tours/Tours.tsx [hunk 3]
```diff
@@ -1,11 +1,68 @@
...
+    imageURL: string;
+    isOpen: boolean;
+    name: string;
+    id: number;
+};
 
-const Tours: React.FC<{}> = () => (
-    <ScreenTitle>{translate('navigation.tours')}</ScreenTitle>
-);
+interface ITour {
```
Comment (NelsonPRSousa) [inline src/components/screens/tours/Tours.tsx]: Are you planning to use this interface?

File: src/components/screens/tours/tour-item/TourItem.tsx (hunk 2 above)
Comment (joaopedrodcf) [reply src/components/screens/tours/tour-item/TourItem.tsx]: Added the translation of the title that is on the routes.js

The rest of the mocks I don't think makes sense translate on client side as this gonna come from the API

File: src/components/screens/tours/Tours.tsx (hunk 3 above)
Comment (joaopedrodcf) [reply src/components/screens/tours/Tours.tsx]: Yeah now I'm using it thx

**Pull request merged**